stereovision/calibration.py

```diff
@@ -222,17 +222,18 @@ def calibrate_cameras(self):
         flags = (cv2.CALIB_FIX_ASPECT_RATIO + cv2.CALIB_ZERO_TANGENT_DIST +
                  cv2.CALIB_SAME_FOCAL_LENGTH)
         calib = StereoCalibration()
+        print(self.image_size, len(self.object_points), len(self.image_points["left"]), len(self.image_points["right"]))
```

Comment:
I don't like printing in anything that's not a CL-util; instead, I prefer using ``logging`` and throwing it out at the ``logging.INFO`` level - it allows client code to redirect it wherever they want, which might be the command line if that's where they'd like it. Would you mind putting this in a call to ``logging.INFO``?